lib/semantics/check-stop.cc

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2019, Arm Ltd.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "check-stop.h"
+#include "semantics.h"
+#include "tools.h"
+#include "../common/Fortran.h"
+#include "../evaluate/expression.h"
+#include "../parser/parse-tree.h"
+#include <optional>
+
+Fortran::semantics::StopChecker::StopChecker(
+    Fortran::semantics::SemanticsContext &context)
```

Comment:
Please follow the pattern of the other .cc files and wrap it all in `namespace Fortran::semantics`. Then you can shorted all of the names starting with `Fortran::`.

Reply:
Done.